promitor-agent-scraper/README.md

```diff
@@ -61,6 +61,7 @@ their default values.
 | `image.tag`  | Tag of image to use | None, chart app version is used by default            |
 | `image.pullPolicy`  | Policy to pull image | `Always`            |
 | `image.pullSecrets`  | ImagePullSecrets for the pod | `[]`            |
+| `concurrency.mutexTimeoutSeconds` | A scrape job's maximum time hold(in seconds) a scraping mutex. | `90`
```

Comment:
Can you please move this after azureAuthentication to follow the alphabetical ordering?